spec/ad_parser.spec.js

```diff
@@ -0,0 +1,37 @@
+import { parseAd } from '../src/parser/ad_parser';
```

Comment:
Which duplicated files? the xml or the unit tests? The test folder is deprecated and we will delete everything in it when all the unit tests have been moved. Meanwhile I still need the xml files to test in the new jest stack, so that is why I'm duplicating them for now.